# Imu to Depth
T_I_D:
  quaternion: [-0.9999835714054479,  0.004912506370950757,  0.001848877977980229,  0.002303443201699748]  #[0.7106305897844819, 0.70355963864234772, -0.0028283804568536592, 0]
  translation_m: [0.03588924870877858,  -0.007040485471323275,  -0.002579107122546876] #[-0.0066278342814687129, 0.03607990536312302, 0.0028830958578366923]

# Imu to fisheye Tango
T_T_F:
  quaternion: [0.9999726766543754,  -0.007298695470581126,  -0.0009855696825188187,  0.0006353278782214075]
  translation_m: [-0.002357223151009965,  -0.005195546543320844,  -0.004493632750266145]

# Imu to fisheye Rovioli
T_R_F:
  matrix: [0.015312935197755421, 0.9998820897725589, 0.0011491595578884215,0.0006476368942388009, 0.9998810666840783, -0.015310790326910325, -0.0018526160624660166, 0.0015771952779394993, -0.0018348030790417361, 0.0011773918742425417, -0.9999976236201941, -0.004660912746997629, 0.0, 0.0, 0.0, 1.0]

T_W_I:
  quaternion: [-0.03096250521, 0.453607236, -0.886581038, 0.08518134599]
  translation_m: [0.09680868996, 0.1081281501, -0.3696407169]

T_I_D:
  quaternion: [0.999983571405, -0.00491250637095, -0.00184887797798, -0.0023034432017]
  translation_m: [0.0358892487088, -0.00704048547132, -0.00257910712255]

T_M_I:
  quaternion: [0.110691974494, -0.422534513637, 0.277605928622, 0.855655783573]
  translation_m: [-0.433757300467, -0.365367944187, -0.189360287351]
